src/main/groovy/com/okta/test/mock/wiremock/TestUtils.groovy

```diff
@@ -46,6 +56,39 @@ class TestUtils {
         return resizedBytes;
     }
 
+    static ExtractableResponse followRedirectUntilLocation(ExtractableResponse response,
+                                                           Matcher<Response> responseMatcher,
+                                                           int maxRedirects = 3) {
+
+        if (responseMatcher.matches(response)) {
+            return response
+        }
+
+        for (int ii=0; ii<maxRedirects; ii++) {
+
+            def location = response.header("Location")
+            println("redirecting to:  ${location}")
+
+            assertThat "Location is null, could not match or follow redirect", location, notNullValue()
+
+            def tempResponse = given()
+                .redirects()
+                    .follow(false)
+                .accept(ContentType.JSON)
+            .when().log().everything()
+                .cookies(response.cookies())
+                .get(location)
+            .then().log().everything()
+            .extract()
+
+            response = tempResponse
```

Comment:
Is this line needed? We should be able to set this above:

```java
response = given()
                .redirects()
                    .follow(false)
                .accept(ContentType.JSON)
            .when()
                .log()
                .everything()
                .cookies(response.cookies())
                .get(location)
            .then()
                .log()
                .everything()
                .extract()
```

Reply:
i'll pull those out, the log() directive is one of the items i don't like about this lib, i wish there was just a logger you could enable.  Though you can configure some of the defaults, so maybe there is a way 🤔